st2actions/st2actions/scheduler/handler.py

```diff
@@ -62,6 +61,9 @@ def __init__(self):
         self.message_type = LiveActionDB
         self._shutdown = False
         self._pool = eventlet.GreenPool(size=cfg.CONF.scheduler.pool_size)
+        self._execution_scheduling_timeout_threshold_min = \
```

Comment:
This variables should named `self._execution_scheduling_timeout_threshold_ms` so it's `ms` for milliseconds and not `min` for minutes.

Reply:
@guzzijones Can you hook this up?